geostreams/geodashboard
# Add regions trends views

## Description
This PR adds two new views: regions trends and their detail views. This required some changes to the Sensor popup container so it can be used by trends too.

### How to test:
First, install the new dependencies by running `yarn` in the `packages/geostreaming` folder.

You can then use your own local geostreams instance with GLM database, or use the production database by running `GEOSTREAMS_URL=https://greatlakesmonitoring.org/geostreams yarn run geostreaming`.

You can start exploring the new views by going to `localhost:8080/trends/regions` and following the links there. Use this view on the production site for reference: https://greatlakesmonitoring.org/trendsregions.

Another way to test is through this [PR](https://github.com/geostreams/glm/pull/6) on GLM repo, which depends on changes here. This way, you can review both PRs at the same time! Just pull the new changes for the GLM repo and start it by running `GEOSTREAMS_URL=https://greatlakesmonitoring.org/geostreams yarn start`. You can then investigate the views as mentioned above. The explore page also needs to be reviewed to make sure nothing has broken there. The GLM PR replaces the old explore page with the new one in the `geostreaming` package.

### Issue link:
Resolves #52

## Review Time Estimate


## Types of changes (select all that applies)


## Checklist:


- [x] This changeset requires updating dependencies.
